### DISCUSSION TITLE
AttributeError: module 'meta.settings' has no attribute 'FB_APPID'

### DISCUSSION THREAD

**Discussion by dxrvip** [django-cms/django-cms#7656] [Q&A] [answered]
#0 0.611     return _bootstrap._gcd_import(name[level:], package, level)
#0 0.611   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
#0 0.611   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
#0 0.611   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
#0 0.611   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
#0 0.611   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
#0 0.611   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
#0 0.611   File "/usr/local/lib/python3.9/site-packages/djangocms_blog/models.py", line 29, in <module>
#0 0.611     from .cms_appconfig import BlogConfig
#0 0.611   File "/usr/local/lib/python3.9/site-packages/djangocms_blog/cms_appconfig.py", line 10, in <module>
#0 0.611     from .settings import MENU_TYPE_COMPLETE, get_setting
#0 0.611   File "/usr/local/lib/python3.9/site-packages/djangocms_blog/settings.py", line 198, in <module>
#0 0.611     BLOG_FB_APPID = meta_settings.FB_APPID
#0 0.611 AttributeError: module 'meta.settings' has no attribute 'FB_APPID'

**Comment by fsbraun** [comment, marked as answer]
Seems to be raised by [djangocms-blog](https://github.com/nephila/djangocms-blog). Maybe ask there?

**Comment by dxrvip** [comment]
ok

**Comment by corentinbettiol** [comment]
tl;dr:

- stay on `django-meta==2.3.0` if you're still using django <= 2
- update django-meta and some packages (like djangocms-blog, to 2.0.4+) if you're using django 3+
